packages/cli/src/types.ts

```diff
@@ -59,8 +59,11 @@ export type FixCommandOptions = {
   ignore: string[];
   graph?: boolean;
   mode: 'single-pass' | 'drain';
+  skipChecks?: SkipChecks;
 };
 
+export type SkipChecks = ('eslint' | 'deps')[];
```

Comment:
// list of all the allowed checks which can be skipped
export enum SKIP_CHECKS {
  eslint = 'eslint',
  deps = 'deps',
}

// create the type from the enum
export type SkipChecks = (keyof typeof SKIP_CHECKS)[] | undefined;

Reply:
Done